app/views/shared/_error_messages.html.erb

```diff
@@ -3,7 +3,7 @@
 <% if object.errors.any? %>
   <div id="error_explanation">
     <div class="alert alert-danger">
-      The form contains <%= pluralize(object.errors.count, "error") %>.
+      error_messages.the_form_contains <%= pluralize(object.errors.count, "error") %>.
```

Comment:
The Rails solution is here: https://robots.thoughtbot.com/pluralizing-rails-i18n-translations. It can be implemented like this:
```ruby
<%= t 'error_messages.form_contains', count: object.errors.count %>
```